### PR TITLE
Fixed #2655 snackbar added when albums added/removed from excluded

### LINKED ISSUE #2655: Display SnackBar upon adding / Removing folders to exclude list
**Describe the bug**
Currently no snackbar is shown upon adding removing albums from exclude list.

**To Reproduce**
1. Add/Remove albums from exclude 

**Expected behavior**

Snackbar should be there to indicate the activity with exclude list

**Smartphone Info:**

|               | |
|---------------|-|
|Device         | Moto G5s Plus |
|Android Version| Oreo 8.1 |

**Would you like to work on the issue?**

- [x] Yes
- [ ] No
- Other:

### PR DESCRIPTION
**Pull request by neel1998**
Fixed #2655 

Changes:Added snackbar to indicate if album/s is/are added or removed from excluded items

Screenshots of the change: 
screenshot_20190227-104647
screenshot_20190227-104622

### CODE REVIEW

**Comment by yashk2000** [comment]
Please remove the unnecessary imports.

**Comment by yashk2000** [comment]
Please remove the unnecessary imports.

**Comment by neel1998** [comment]
@yashk2000  i have removed the imports, please review

File: app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -661,6 +661,8 @@
         By excluding this folder you will also exclude all sub-folders. Below you can find a list
         that allows you to choose a parent, if you have to exclude more folders which are on the same level.
     </string>
+    <string name="exclude_album_snackbar_message">Selected Item/s excluded</string>
```
Comment (abishekvashok) [inline app/src/main/res/values/strings.xml]: It should be `item(s)`

Comment (neel1998) [reply app/src/main/res/values/strings.xml]: Okay... I will make the change.

**Comment by mariobehling** [comment]
@neel1998 Please finalize this PR. So, we can move on. Thank you!

**Comment by neel1998** [comment]
@mariobehling  @yashk2000  i have resolved the conflicts . Kindly review.

File: app/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -666,6 +666,8 @@
         By excluding this folder you will also exclude all sub-folders. Below you can find a list
         that allows you to choose a parent, if you have to exclude more folders which are on the same level.
     </string>
+    <string name="exclude_album_snackbar_message">Selected item(s) excluded</string>
+    <string name="unExclude_album_snackbar_message">Selected item(s) removed from excluded</string>
```
Comment (yashk2000) [inline app/src/main/res/values/strings.xml]: Make it "Selected item(s) are no longer excluded".

**Comment by neel1998** [comment]
@yashk2000  i have made the changes. Review once

**Comment by abishekvashok** [comment]
Thanks

**Pull request merged**